### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by johan12345** [DaRasch/spiceminer#6]
Is this supposed to be compatible with Python 3?
I am getting many errors in `_spicewrapper.py`, for example `ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type` (with can be solved by using `.encode('ascii')` on string arguments

**Comment by DaRasch** [comment]
No, python3 support is planned but not a top priority at the moment.
Did I mention python3 support anywhere in the docs? I hope not.

**Comment by johan12345** [comment]
No, you just didn't mention that it is not supported, either ;)
